iset.mm

```diff
@@ -81880,6 +81880,53 @@ Ordering on reals (cont.)
   0le1 $p |- 0 <_ 1 $=
     ( cc0 c1 0re 1re 0lt1 ltleii ) ABCDEF $.
 
+  ${
+    $d x B $.  $d x y C $.  $d x y D $.  $d x y M $.  $d x y N $.  $d x y ph $.
+    $d x y S $.
+    ltord.1 $e |- ( x = y -> A = B ) $.
+    ltord.2 $e |- ( x = C -> A = M ) $.
+    ltord.3 $e |- ( x = D -> A = N ) $.
+    ltord.4 $e |- S C_ RR $.
+    ltord.5 $e |- ( ( ph /\ x e. S ) -> A e. RR ) $.
+    ${
+      ltord.6 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x < y -> A < B ) ) $.
+      $( Lemma for ~ eqord1 .  (Contributed by Mario Carneiro, 14-Jun-2014.) $)
+      ltordlem $p |- ( ( ph /\ ( C e. S /\ D e. S ) ) ->
+                        ( C < D -> M < N ) ) $=
+        ( clt wbr wi wceq cv wral wcel wa ralrimivva breq1 breq1d imbi12d breq2
+        eqeq1 eqeq1d chvarv breq2d rspc2v mpan9 ) ABUAZCUAZQRZDEQRZSZCHUBBHUBFH
+        UCGHUCUDFGQRZIJQRZSZAUTBCHHPUEUTVCFUQQRZIEQRZSBCFGHHUPFTZURVDUSVEUPFUQQ
+        UFVFDIEQLUGUHUQGTZVDVAVEVBUQGFQUIVGEJIQUPGTZDJTZSVGEJTZSBCUPUQTZVHVGVIV
+        JUPUQGUJVKDEJKUKUHMULUMUHUNUO $.
+
+      $( Infer an ordering relation from a proof in only one direction.
+         (Contributed by Mario Carneiro, 14-Jun-2014.)  (Revised by Jim
+         Kingdon, 20-Dec-2022.) $)
+      eqord1 $p |- ( ( ph /\ ( C e. S /\ D e. S ) ) -> ( C = D <-> M = N ) ) $=
+        ( wcel wa wceq cr cv wex simprl elisset syl adantr adantl biimpa eqtr3d
+        wb eqeq2 exlimddv ex clt wn ltordlem con3d wi ancom2s anim12d ralrimiva
+        wral eleq1d rspccva sylan adantrr adantrl lttri3d sseldi simprr 3imtr4d
+        wbr impbid ) AFHQZGHQZRRZFGSZIJSZVPVQVRVPVQRZBUAZFSZVRBVPWABUBZVQVPVNWB
+        AVNVOUCZBFHUDUEUFVSWARZDIJWADISVSLUGWDVTGSZDJSVSWAWEVQWAWEUJVPFGVTUKUGU
+        HMUEUIULUMVPIJUNVLZUOZJIUNVLZUOZRFGUNVLZUOZGFUNVLZUOZRVRVQVPWGWKWIWMVPW
+        JWFABCDEFGHIJKLMNOPUPUQAVOVNWIWMURAVOVNRRWLWHABCDEGFHJIKMLNOPUPUQUSUTVP
+        IJAVNITQZVOADTQZBHVBZVNWNAWOBHOVAZWOWNBFHWADITLVCVDVEVFAVOJTQZVNAWPVOWR
+        WQWOWRBGHWEDJTMVCVDVEVGVHVPFGVPHTFNWCVIVPHTGNAVNVOVJVIVHVKVM $.
+    $}
+
+    ltord2.6 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x < y -> B < A ) ) $.
+    $( Infer an ordering relation from a proof in only one direction.
+       (Contributed by Mario Carneiro, 14-Jun-2014.) $)
+    eqord2 $p |- ( ( ph /\ ( C e. S /\ D e. S ) ) -> ( C = D <-> M = N ) ) $=
```

Comment:
Same as above with "strictly decreasing" ?

Reply:
I've pushed up a change to use what is basically BJ's wording, for both set.mm and iset.mm. If we want to revise further we can always do so later.